### ISSUE TITLE
Support multipart/form-data --> POST /send-results

### ISSUE THREAD

**Issue by fescobar** [fescobar/allure-docker-service#81]
To avoid encoding files in base64, it would be good if we implement  **"multipart/form-data"** to load the files directly. Like this:

```
curl -X POST 'http://localhost:5050/send-results?project_id=default' -H 'Content-Type: multipart/form-data' -F 'files[]=@/my-dir/allure-results/my-file-01.json' -F 'files[]=@/my-dir/allure-results/my-file-02.json' -ik
```

**Comment by fescobar** [comment]
@KaylaCronk I've released a beta version with this feature added `image: "frankescobar/allure-docker-service:beta"`. Remove a previous local `beta` if you have one.

The documentation is up to date in `beta` branch.
https://github.com/fescobar/allure-docker-service/tree/beta#send-results-through-api

In swagger UI there is an issue related to the new content type. I'm waiting for this release to have that working: https://github.com/sveint/flask-swagger-ui/issues/32

You can use this example sending multiple files in a simple request with curl: https://github.com/fescobar/allure-docker-service/blob/beta/allure-docker-api-usage/send_results.sh


Please, test it and let me know if there is a problem or not.

**Comment by fescobar** [comment]
@KaylaCronk I've re-released `2.13.3` including this feature with examples. Please, remove your previous `2.13.3` if you have one.
